Source/RunActivity/Viewer3D/Scenery.cs

```diff
@@ -341,19 +341,23 @@ public WorldFile(Viewer viewer, int tileX, int tileZ, bool visible)
                     }
                 }
 
-                if (shapeFilePath != null && File.Exists(shapeFilePath + "d"))
+                var shapeDescriptorPath = shapeFilePath + "d";
+                if (shapeFilePath != null)
```

Comment:
This is no longer checking if the SD file exists before loading it, which causes crashes for the many situations where someone's deleted SD files because they were told they weren't needed.

Reply:
I believe it's fixed now, so I've removed the label not-for-unstable.